storage/fsStorage.go

```diff
@@ -1,20 +1,71 @@
 package storage
 
-import "fmt"
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"os"
+)
 
-type FsStorage struct {
+type fsStorage struct {
 	path string
 }
 
-func (st FsStorage) Read(key string) (string, error) {
-	return "got it", nil
+func (st fsStorage) Read(key string) (string, error) {
+	file, err := ioutil.ReadFile(st.path)
+	if err != nil {
+		return "", err
+	}
+	data := make(map[string]string)
+	json.Unmarshal(file, &data)
+	val, ok := data[key]
+	if ok {
+		return val, nil
+	}
+	return "", errors.New("item was not found")
 }
-func (st FsStorage) Write(key string, value string) {
-	fmt.Print(value)
+
+func (st fsStorage) Write(key string, value string) error {
+	err := checkFile(st.path)
+	if err != nil {
+		return err
+	}
+
+	file, err := ioutil.ReadFile(st.path)
+	if err != nil {
+		return err
+	}
+	data := make(map[string]string)
```

Comment:
As I remember we agreed on the next file format:

```json
{
    "key1": {"value": "val1"},
    "key2": {"value": "val2"},
}
```

Reply:
refactored in the [last commit](https://github.com/retttddd/golangProjectv3/commit/20b494fbae8da0720f5baff99d9d6b9c59bb0d9a)